.changeset/tidy-beers-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/etherchain-adapter': minor
+---
+
+Updated etherchain gasprice endpoint to use /api/gasnow api. Updated tests to match new endpoint
```

Comment:
This branch has been updated automatically and is now ahead of where it should be. You need to revert the latest commit(s)